azurerm/internal/services/apimanagement/api_management_resource.go

```diff
@@ -606,24 +610,41 @@ func resourceArmApiManagementServiceRead(d *schema.ResourceData, meta interface{
 		return fmt.Errorf("making Read request on API Management Service %q (Resource Group %q): %+v", name, resourceGroup, err)
 	}
 
-	signInClient := meta.(*clients.Client).ApiManagement.SignInClient
-	signInSettings, err := signInClient.Get(ctx, resourceGroup, name)
-	if err != nil {
-		return fmt.Errorf("retrieving Sign In Settings for API Management Service %q (Resource Group %q): %+v", name, resourceGroup, err)
-	}
+	if resp.VirtualNetworkType != apimanagement.VirtualNetworkTypeInternal {
```

Comment:
this needs to use `resp.[XX]Properties.VirtualNetworkType` rather than the top-level property